EIPS/eip-7800.md

```diff
@@ -0,0 +1,220 @@
+---
+eip: 7800
+title: Withdrawal Credential Update Request
+description: Allow validators to update their withdrawal credentials via execution requests
+author: lucassaldanha (@lucassaldanha)
+discussions-to: https://ethereum-magicians.org/t/eip-7800-withdrawal-credential-update-request/21514
+status: Draft
+type: Standards Track
+category: Core
+created: 2024-10-31
+---
+
+## Abstract
+
+This proposal defines a mechanism to allow validators to update their withdrawal
+credentials using a new execution request type (0x03). The request allows for
+changing the execution address and the withdrawal credential prefix (0x01 or 0x02).
+
+## Motivation
+
+When the ability to update a validator BLS withdrawal credentials to execution
+address was introduced in Capella, one of the most common questions was about
+allowing the withdrawal credential to be changed in the future.
+Either for security (e.g. credential rotation) or to allow for alternative ways
+of handling withdrawals (e.g. having a contract address as credentials).
+The main reason for not adding this options was because the complexity of implementing
+this communication channel between the Execution Layer and the Consensus Layer was
+complicated (based on the experience with the Eth1 bridge).
```

Comment:
```suggestion
The main reason for not adding this options was because implementing
this communication channel between the Execution Layer and the Consensus Layer was
complex (based on the experience with the Eth1 bridge).
```
I'd suggest rephrasing this